### PR TITLE
Google Maps all team paths

### PR DESCRIPTION
**Pull request by thepiwo**

### CODE REVIEW

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -27,6 +27,9 @@
         </intent-filter>
       </activity>
       <activity android:name="com.facebook.react.devsupport.DevSettingsActivity" />
+        <meta-data
+                android:name="com.google.android.geo.API_KEY"
+                android:value=""/>
```
Comment (florianschmidt1994) [inline android/app/src/main/AndroidManifest.xml:32]: Maybe we can get this in js config

File: app/locations/actions.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    return dispatch => {
+        withAccessToken(api, store.getState())
+            .getAllEvents()
+            .then(events => {
+                const currentEvents = events.filter(e => e.isCurrent);
+                return Promise.all(currentEvents.map(event => api.fetchLocationsForEvent(event.id)))
+                    .then(eventLocations => {
+                        let teams = [];
+                        const locations = [].concat(...eventLocations);
+                        locations.forEach(tl => {
```
Comment (florianschmidt1994) [inline app/locations/actions.js]: Move to function?

File: app/screens/MapScreen.js [hunk 3]
```diff
@@ -1,14 +1,204 @@
 import React, {Component} from 'react';
-import {Text, View} from "react-native";
+import {StyleSheet, View} from "react-native";
 import {Icon} from "native-base";
+import MapView, {Polyline, Marker} from 'react-native-maps';
+import {connect} from "react-redux";
+import {fetchLocations} from "../locations/actions";
 
-export default class MapScreen extends Component {
+const mapStyle = [
```
Comment (florianschmidt1994) [inline app/screens/MapScreen.js:8]: move to file?

File: app/screens/MapScreen.js [hunk 4]
```diff
@@ -1,14 +1,204 @@
...
         drawerIcon: () => <Icon name='map'/>
     };
 
+    componentDidMount() {
+        this.props.onRefresh()
+    };
+
     render() {
-        return <View><Text>Map will be here</Text></View>
+        let teamMarkers = this.props.locations.map(team => {
```
Comment (florianschmidt1994) [inline app/screens/MapScreen.js]: const? Also below

**Review by florianschmidt1994 — COMMENTED** [review]
Added some comments. Looks pretty good in general!

File: app/screens/MapScreen.js (hunk 3 above)
Comment (thepiwo) [reply app/screens/MapScreen.js:8]: nope, will only use here

File: android/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (thepiwo) [reply android/app/src/main/AndroidManifest.xml:32]: possible with react-native-config, but not in scope of this task

**Pull request merged**